.travis.yml

```diff
@@ -91,6 +91,8 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ./python Tools/scripts/patchcheck.py --travis $TRAVIS_PULL_REQUEST; fi
   # `-r -w` implicitly provided through `make buildbottest`.
   - make buildbottest TESTOPTS="-j4 -uall,-cpu"
+  # Check if all symbols exported by libpython start with "Py" or "_Py"
```

Comment:
Maybe change "Check if" to "Check that" or "Validate that" for better clarity?

Reply:
done